README.md

```diff
@@ -38,6 +38,7 @@ Examples are Cortex-M55 or Cortex-M85 configured with MVE.
 | Softmax         | Yes         | Yes        | N/A        | Yes         | Yes          | N/A          | Yes         | No           | N/A          |
 | LSTM            | Yes         | Yes        | No         | Yes         | Yes          | No           | Yes         | Yes          | No           |
 | SVDF            | Yes         | No         | No         | Yes         | No           | No           | Yes         | No           | No           |
+| Pad             | Yes         | No         | No         | Yes         | No           | No           | Yes         | No           | No           |
```

Comment:
Should this be N/A instead of No as no weights

Reply:
I think no is the right one here, because it is possible to have int16 padding for example but we do not have the support for it? It seems like this table is not intended for weights but rather base operator support

Reply:
int16 padding would mean input and output in int16, right? That is second column with a No already